.github/workflows/reusable-workflow.yml

```diff
@@ -89,13 +100,25 @@ jobs:
           CYPRESS_BIDDER_MNEMONIC_INPUT: ${{ inputs.bidder_mnemonic }}
           CYPRESS_BIDDER_ADDRESS_INPUT: ${{ inputs.bidder_address }}
 
-      - name: Archive e2e artifacts
+      - name: Archive e2e artifacts locally
         uses: actions/upload-artifact@v3
```

Comment:
_low priority_

Bump this action to v4 as v3 will be deprecated in half a year?

Reply:
You're right. Done